### PR TITLE
Remove mentions of ~

### PR DESCRIPTION
**Pull request by Dawnofares**
one last one, I think

### CODE REVIEW

**Comment by AnnikaCodes** [comment]
This should probably also be fixed in all the rules translations.

**Comment by Dawnofares** [comment]
right, I'll get on that rq

**Comment by Dawnofares** [comment]
Weird, it looks like the Japanese one has RO, should that be changed for all?

**Comment by AnnikaCodes** [comment]
> Weird, it looks like the Japanese one has RO, should that be changed for all?

Probably?

File: website/pages/rules-pl.md [hunk 1]
```diff
@@ -32,12 +32,12 @@
 
 Twoja nazwa użytkownika może być wybrana i zmieniona w dowolnym czasie (przez wylogowanie i zalogowanie z nową nazwą). Pamiętaj że:
 
-1. **Nazwy użytkowników nie mogą podszywać się** pod znaną osobę (`%`, `@`, `&` lub `~` na lewo od nazwy) czy sławnej osoby/organizacji która używa PS albo jest powiązana z serią Pokémon.
+1. **Nazwy użytkowników nie mogą podszywać się** pod znaną osobę (`%`, `@`, lub `&` na lewo od nazwy) czy sławnej osoby/organizacji która używa PS albo jest powiązana z serią Pokémon.
```
Comment (xfix) [inline website/pages/rules-pl.md:35]:
```suggestion
1. **Nazwy użytkowników nie mogą podszywać się** pod znaną osobę (`%`, `@` lub `&` na lewo od nazwy) czy sławnej osoby/organizacji która używa PS albo jest powiązana z serią Pokémon.
```

**Comment by Zarel** [comment]
The Japanese one including `#` makes sense to me. Impersonating ROs is probably bad.

**Comment by Dawnofares** [comment]
alright, It should be ready

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**